src/test/java/teammates/test/cases/browsertests/InstructorEditStudentFeedbackPageUiTest.java

```diff
@@ -67,6 +67,7 @@ private void testEditResponse() throws Exception {
 
         submitPage.clickSubmitButton();
         submitPage.verifyStatus(Const.StatusMessages.FEEDBACK_RESPONSES_SAVED);
+        submitPage.waitForAndDismissAlertModal();
```

Comment:
Could create a method in FeedbackSubmitPage to verify that the correct alert modal (message, style class) is created.